### DISCUSSION TITLE
Installing Spike 3 Hub OS on Robot Inventor

### DISCUSSION THREAD

**Discussion by aivarannamaa** [pybricks/support#1115] [General]
code.pybricks.com has "Restore official LEGO firmware" feature. Does this mean you have latest firmware for Spike Prime and Robot Inventor available on your server? Or do you use a public resource on LEGO servers? Can you share the URL?

I've heard that with Pybricks tools (pybricksdev?) it's possible to install the latest Spike 3 firmware on the Robot Inventor. I'd like to try that, but I don't have the firmware.

Even better -- now that LEGO has abandoned Robot Inventor, could you add the possibility to install Spike firmware on Robot Inventor right from your webapp?

**Comment by laurensvalk** [comment]
It would be easy to do, but at the moment we only have permission from LEGO Education to use the V2 files for restoring the firmware.
